testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeJmx.java

```diff
@@ -38,7 +38,7 @@ public void testJmxTablesExposedByDeltaLakeConnectorBackedByGlueMetastore()
                 row("io.trino.plugin.hive.metastore.cache:name=delta,type=cachinghivemetastore"),
                 row("io.trino.plugin.hive.metastore.glue:name=delta,type=gluehivemetastore"),
                 row("io.trino.plugin.hive.metastore.glue:name=delta,type=gluemetastorestats"),
-                row("io.trino.plugin.hive:catalog=delta,name=delta,type=fileformatdatasourcestats"),
+                row("io.trino.plugin.base.metrics:catalog=delta,name=delta,type=fileformatdatasourcestats"),
```

Comment:
this change wan't required for oss delta lake pt but observed that failure in parent PR. https://github.com/trinodb/trino/actions/runs/12152695165/job/33889755245?pr=24359#step:7:4193